cpu/cc26x0/periph/rfc.c

```diff
@@ -72,13 +58,54 @@ void rfc_irq_disable(void)
     NVIC_EnableIRQ(RF_HW_IRQN);
 }
 
+uint32_t rfc_send_cmd(uint32_t ropAddr)
+{
+    RFC_DBELL->CMDR = ropAddr;
+  /* wait for cmd ack (rop cmd was submitted successfully) */
+  while(RFC_DBELL->RFACKIFG<<31);
+  while(!RFC_DBELL->CMDSTA);
+  if (RFC_DBELL->CMDSTA == 1)
+  {
+    printf("%s: radio operation command acknowledged. \n",__FUNCTION__);
```

Comment:
`. ` shouldn't be necessary
i'm not a huge fan of commiting debug.h-stuff in here :-/